### ISSUE TITLE
[Feature] Improve the site Responsiveness

### ISSUE THREAD

**Issue by Marvisjaja** [creativecommons/search#88]
## Problem
The site responsiveness is already made available via my check on the code base  (source code), but from my observation it isn't at least 100% Responsive, i checked this using a screen size starting from 300px  X  600px,  (widthpx X heightpx), then went down to 255px X 600px and noticed that the page body went out of device width.

![cc-search](https://user-images.githubusercontent.com/115430180/195773661-9230426f-33f8-408b-b9e1-240c06841967.PNG)
![cc-search2](https://user-images.githubusercontent.com/115430180/195773671-28d1fdc1-17e7-4e3d-af10-56f3aa1d9b86.PNG)

## Description
An improvement is highly required, reason is because, users over the world have various sizes of device, users with low screen size would have a limited visual on the page content.

## Alternatives
Working efficiently on the code base to improve the page responsiveness would be a fine solution, after this process, the page in question would take the actual width and height of whatever size of device it is being accessed with, in a well and organized form

## Additional context
It is seen that the current responsiveness on the page isn't on a rating of 100% and needs urgent improvements so as to satisfy users of all kinds. A very clear example is the github page below, i reduced the screensize to 300px X 600px and the page could comfortably contain my current screensize. To check this, use google inspect feature.
![pagegit](https://user-images.githubusercontent.com/115430180/195780435-ab8428f9-5483-4d0d-a514-5f9c4c8bfa57.PNG)
Page url: https://github.com/creativecommons/search/issues

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by mbonamensa** [comment]
Hi, @Marvisjaja @possumbilities  I'm interested in helping resolve this if that's okay.

**Comment by PreciousOritsedere** [comment]
@possumbilities Can I work on this?

**Comment by Marvisjaja** [comment]
Hi, good day @possumbilities , I'm still waiting for a (Positive) Response, as I'm highly interested to rectify and resolve this issue

**Comment by possumbilities** [comment]
I agree this is an Issue. However, I'm moving this to `status: blocked` since a lot of other design work needs to happen before this is addressed in a measurable way. Thanks, so much for documenting it here!

**Comment by Marvisjaja** [comment]
Good evening sir/Ma, please i need to know, since you're moving this issue to status blocked because of other designs that need to be in place, do i still need to bring  this up as a contribution again or you'll have to remove the block status when the time is right.In essence what I'm trying to mean is, will i still have the privilege to work on this later or since the status  is blocked i won't be able to work on it anymore.
Lastly, can i create an entirely different contribution on another issue while this one has a blocked status?

Sent from Yahoo Mail for iPhone


On Friday, October 14, 2022, 9:26 PM, possumbilities ***@***.***> wrote:




I agree this is an Issue. However, I'm moving this to status: blocked since a lot of other design work needs to happen before this is addressed in a measurable way. Thanks, so much for documenting it here!

**Comment by possumbilities** [comment]
@Marvisjaja Hi! To help clarify: creating the Issue counts as a contribution, whether it can be worked on via a PR or not. If later on it becomes unblocked it can be worked on again.

I'd refer you to the [Contribution Guidelines](https://opensource.creativecommons.org/contributing-code/#finding-an-issue) on what labels indicate an Issue is able to be worked on. 

> Issues not ready for work:
The following tags mark issues that are not open for community contribution:
🚧 status: blocked: Blocked by other work that needs to be done first
🧹 status: ticket work required: Needs additional work before it is ready to be taken up
🚦 status: awaiting triage: Has not been triaged by a maintainer
Do not work on these.
